api_tests/ledger_query_service/test.js

```diff
@@ -373,14 +373,31 @@ describe("Test Ledger Query Service API", () => {
             // keccak('Transfer(address,address,uint256)')
             const transfer_topic =
                 "0xddf252ad1be2c89b69c2b068fc378daa952ba7f163c4a11628f55a4df523b3ef";
+            const from_address =
+                "0x000000000000000000000000" +
+                alice_wallet_address.replace("0x", "");
+            const to_address =
+                "0x00000000000000000000000005cbb3fdb5060e04e33ea89c6029d7c79199b4cd";
+
             let location;
             it("LQS should respond with location when creating a valid transaction receipt query", async function() {
                 this.timeout(1000);
                 return chai
                     .request(lqs.url())
                     .post("/queries/ethereum/logs")
                     .send({
-                        topics: [[transfer_topic]],
+                        event_matchers: [
+                            {
+                                address: token_contract_address,
+                                data:
+                                    "0x0000000000000000000000000000000000000000000000000000000000000001",
```

Comment:
:question: Is this `data` the transaction payload?

Reply:
It's part of the log message